roles/wisdom_deploy_model/tasks/main.yml

```diff
@@ -0,0 +1,146 @@
+
+- name: Patch the service account with the quay pullsecret
+  shell: |
+    set -o pipefail
+    oc patch serviceaccount modelmesh-serving-sa -n {{ wisdom_deploy_model_namespace }} -p '{"imagePullSecrets": [{"name": "quay-io"}]}'
+    oc delete pods --all --force -n {{ wisdom_deploy_model_namespace }}
+  
+
+# Wait for wisdom Pods to be ready. 
+# Max time is 15 minutes to give time for the GPU operator stack and image pull to complete.
+- name: Wait for wisdom model mesh Pods to be ready
+  block:
+  - name: Ensure that wisdom model mesh deployment is ready
+    command:
+      oc get deployment/modelmesh-serving-{{ wisdom_deploy_model_serving_runtime_name }}
+         -n {{ wisdom_deploy_model_namespace }}
+         -ojsonpath={.status.unavailableReplicas}
+    register: wisdom_number_unavailable
+    until:
+    - wisdom_number_unavailable.rc == 0
+    - not wisdom_number_unavailable.stdout
+    retries: 30
+    delay: 30
+
+# Put wisdom protos in a Secret
+- name: Delete the protos Secret if it already exists
+  command: oc delete secret/wisdom-protos --ignore-not-found -n {{ wisdom_deploy_model_namespace }}
+
+- name: Create the wisdom protos Secret
+  command: oc create secret generic wisdom-protos --from-file={{ wisdom_deploy_model_protos_path }} -n {{ wisdom_deploy_model_namespace }}
+
+- name: Delete the entrypoint ConfigMap if it already exists
+  command: oc delete --ignore-not-found configmap/wisdom-tester-entrypoint -n {{ wisdom_deploy_model_namespace }}
+
+- name: Create the entrypoint ConfigMap
+  command: oc create configmap wisdom-tester-entrypoint --from-file {{ wisdom_tester_entrypoint }} -n {{ wisdom_deploy_model_namespace }}
+
+- name: Define the test image
+  set_fact:
+    wisdom_tester_image: image-registry.openshift-image-registry.svc:5000/{{ wisdom_deploy_model_namespace }}/{{ wisdom_deploy_model_tester_imagestream_name }}:{{ wisdom_deploy_model_tester_image_tag }}
+
+- name: Apply the validator Pod template
+  template:
+    src: "{{ wisdom_tester_pod }}"
+    dest: "{{ artifact_extra_logs_dir }}/src/003_wisdom_tester_pod.yml"
+    mode: 0400
+  
+- name: Delete the wisdom tester Pod if it exists
+  command: oc delete  --ignore-not-found -f "{{ artifact_extra_logs_dir }}/src/003_wisdom_tester_pod.yml" -n {{ wisdom_deploy_model_namespace }}
+
+- name: Create the wisdom tester Pod
+  command: oc create -f "{{ artifact_extra_logs_dir }}/src/003_wisdom_tester_pod.yml" -n {{ wisdom_deploy_model_namespace }}
+
+# Wait for model to respond without error
+- block:
+  - name: Wait for the wisdom test Pod to terminate
+    command:
+      oc get pod/wisdom-tester
+         -n {{ wisdom_deploy_model_namespace }}
+         -o custom-columns=:.status.phase
+         --no-headers
+    register: tester_wait
+    until: tester_wait.stdout == "Succeeded" or tester_wait.stdout == "Error" or tester_wait.stdout == "Failed"
+    retries: 10
+    delay: 30
+
+- name: Ensure that the tester Pod succeeded
```

Comment:
here you'll want to capture the logs of the Pod
they are available in the [gather-extra](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/705/pull-ci-openshift-psap-ci-artifacts-main-wisdom-load-test-light/1663953774378487808/artifacts/load-test-light/gather-extra/artifacts/pods/wisdom_wisdom-tester_wisdom-tester.log), but I strive to provide the answer to the obvious questions in the task artifacts.
Obvious question here is: why did the Pod fail 🙃